bigtable/admin.go

```diff
@@ -1186,12 +1188,16 @@ func (iac *InstanceAdminClient) DeleteCluster(ctx context.Context, instanceID, c
 }
 
 // SetAutoscaling enables autoscaling on a cluster. To remove autoscaling, use
-// UpdateCluster.
+// UpdateCluster. See AutoscalingConfig documentation for deatils.
 func (iac *InstanceAdminClient) SetAutoscaling(ctx context.Context, instanceID, clusterID string, conf AutoscalingConfig) error {
 	ctx = mergeOutgoingMetadata(ctx, iac.md)
 	cluster := &btapb.Cluster{
-		Name:   "projects/" + iac.project + "/instances/" + instanceID + "/clusters/" + clusterID,
-		Config: conf.proto(),
+		Name: "projects/" + iac.project + "/instances/" + instanceID + "/clusters/" + clusterID,
+		Config: &btapb.Cluster_ClusterConfig_{
+			ClusterConfig: &btapb.Cluster_ClusterConfig{
+				ClusterAutoscalingConfig: conf.proto(),
```

Comment:
This will now be a `Cluster_ClusterConfig` object with a nil `ClusterAutoscalingConfig`, rather than just a nil `Cluster_ClusterConfig`. Does that make a difference?

Reply:
I checked -- and both are rejected without a valid autoscaling config.